docker-compose.yml

```diff
@@ -72,23 +72,16 @@ services:
       - "8081:8081"
     cpus: 2
 
-  app:
-    image: node:14-slim
-    working_dir: /usr/local/src
+  react:
+    build:
+      context: ./src/react
+      dockerfile: Dockerfile.local
     volumes:
-      - ./src/app:/usr/local/src
+      - ./src/react:/usr/local/src
       # Use a Docker volume to cache node_modules
-      - node_modules:/usr/local/src/node_modules
-      - ./src/django/static:/usr/local/src/build
-      - $HOME/.aws:/root/.aws:ro
```

Comment:
You completely remove
```
- ./src/django/static:/usr/local/src/build
- $HOME/.aws:/root/.aws:ro
```

Is that necessary and Reat app will work properly without these volumes?

Reply:
cc @Innavin369

Reply:
@VadimKovalenkoSNF @roman-stolar 

>  django static build no longer needed for FE.

Correct.

> aws config

We reviewed the entire project and didn't find a reason to keep it, especially since it's not used in the React app. Furthermore, we confirmed that the React app functions properly after its removal.

Reply:
Ack.